app.rb

```diff
@@ -22,12 +15,31 @@ def list_books
         puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
       end
     end
-    nil
   end
 
-  # list of people method
+  def create_book
+    print 'Title: '
+    book_title = gets.chomp
+    print 'Author: '
+    book_author = gets.chomp
+    book = Book.new(book_title, book_author)
+    @books << book
+    puts 'Book created successfully'
+  end
+
+  def save_books
+    save_to_json('books.json', @books)
+  end
+
+  def load_books
+    load_from_json('books.json', @books)
+  end
+end
```

Comment:
- [ ] When a book is created this is what is saved in the book.json file. Kindly note that the data in the storage file should be human readable, displaying the book attributes, like the title, author and so on, not the memory object.

  ![image](https://github.com/AlyAtalla/oop-library/assets/81468675/3066c82d-6de6-4395-b822-edfdf1543b9e)